internal-tooling/performanceUtils.ts

```diff
@@ -224,3 +225,58 @@ export function getValidationType(): 'md5' | 'crc32c' | boolean | undefined {
     return 'md5';
   }
 }
+
+/**
+ * Converts the supplied test results from javascript objects to cloud monitoring format.
+ *
+ * @param {TestResult[]} results array of test iteration results.
```

Comment:
"results array of test iteration results." doesnt make sense to me without me looking through the code again. can you name the function that should be this input?